.travis.yml

```diff
@@ -11,4 +11,4 @@ cache:
 install: true
 script:
   - cd jaxb-ri
-  - mvn -U -C -Pstaging clean install
\ No newline at end of file
+  - mvn --update-snapshots --strict-checksums -Pstaging,oss-release clean install
```

Comment:
Thanks for the two first, how about `-P`-`--activate-profiles`?

Reply:
So used to to -P, skipped it on autopilot.